scripts/bin/auto-update/utility.sh

```diff
@@ -0,0 +1,105 @@
+termux_pkg_upgrade_version() {
```

Comment:
Wouldn't it make sense to move this to the `scripts/updates` directory?

Reply:
Done